calculator.c

```diff
@@ -0,0 +1,38 @@
+#include<stdio.h>
```

Comment:
Great job - my only suggestion is to add a comment with your name and function of this program since it's good practice.